index.js

```diff
@@ -61,6 +62,44 @@ const SailsHookMongoat = (sails) => ({
     });
   },
 
+  /**
+   * @typedef {object} ShouldCreateIndexInfo
+   * @property {boolean} shouldCreate - Whether or not index updates should be performed.
+   * @property {string} [forbiddenReason] - Reason for which indexes should not be updated.
+   * Must be set when `canCreate` is `false`.
+   */
+  /**
+   * Determines whether or not we are allowed to create or update new indexes, returning
+   * an object with fitting error details in the event where something would prevent our
+   * attempted changes.
+   *
+   * By default, index updates are forbidden when not running in `alter` or `drop` mode,
+   * though this may be overridden by specifying a truthy `mongoat.forceUpdate` value in
+   * the config.
+   *
+   * One convenient way to ensure indexes are updated is to simply pass the force-update
+   * flag when lifting:
+   * `sails console --mongoat.forceUpdate`
+   *
+   * @returns {ShouldCreateIndexInfo}
+   */
+  shouldCreateIndexes() {
+    // If we've mongoat configured to force updates, return immediately indicating we're
+    // allowed.
+    if (sails.config.mongoat && sails.config.mongoat.forceUpdate) {
+      return { shouldCreate: true };
+    }
+    // If we're not in `alter` or `drop` mode, avoid updates.
+    if (sails.config.models && !['alter', 'drop'].includes(sails.config.models.migrate)) {
+      return {
+        shouldCreate: false,
+        forbiddenReason: `Model migration strategy "${sails.config.models.migrate}" was set. Use "alter" or "drop" to enable index creation, or build them manually.`,
```

Comment:
Can you modify the message to say that it can be forced using config? (instead of "build them manually".

Reply:
Added some log about the force option anytime updates are blocked: https://github.com/fpm-git/sails-hook-mongoat/pull/1/commits/62b7592943471367a78e147e9c2bb1e8943a4097